lib/connectors_async.rb

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+require 'connectors_async/job_runner'
```

Comment:
What is the purpose of this file?

Reply:
That's just a file to require all connectors_async classes if developer writes `require connectors_async`